### DISCUSSION TITLE
"My media" and "Sonos favorites" missing, looking for my Spotify lists

### DISCUSSION THREAD

**Discussion by danfor814** [punxaphil/custom-sonos-card#162] [General]
Hello! 

Many thanks for this, it solved my actual problem with grouping rooms and only having one card. Everything works great as long as I start a Spotify playlist in my iphone via the sonos app, i can then switch songs etc. in the sonos card without my phone. But if I am going to use the module itself in HA i cannot see any playlists or "Favorites" to be able to switch list. I have added "favorites" in the Sonos app, so it's not empty in there. I am new to HA and have probably missed something simple here, but I would really appreciate tips on how to get my playlists to show directly into the sonos card. Everything else works as I had hoped, but this part I'm stuck on. Thank you so much!

![1](https://github.com/johanfrick/custom-sonos-card/assets/142484455/f85ae5bf-1ecf-479f-91bf-9f2851469632)
![2](https://github.com/johanfrick/custom-sonos-card/assets/142484455/7f5e5192-3137-497d-ac9e-a975a4a0cbf4)

**Comment by punxaphil** [comment]
I don't think this is a problem with the Sonos Card. Let me know if it looks ok in the built in media browser though.
